### ISSUE TITLE
Low accuracy in verification while finetuning a model

### ISSUE THREAD

**Issue by Magitoneu** [deepinsight/insightface#902]
I am trying to finetune a pretrained model, I can run the command that is mentioned in the readme without any problem. However, it starts with a bad accuracy in train (I can understand that is because the fc7 layer was removed) but it also have a poor accuracy (60%) in verification with the lfw dataset, which should be quite better, because it is using the embedding layer and not the fc7.

Thanks

**Comment by orlevit** [comment]
Following
